js/src/util.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-env mocha */
+'use strict'
+
+const crypto = require('libp2p-crypto')
+const isIPFS = require('is-ipfs')
+
+const chai = require('chai')
+const dirtyChai = require('dirty-chai')
+const expect = chai.expect
+chai.use(dirtyChai)
+util
```

Comment:
`undefined`?

Reply:
That util was supposed to be inside `describe`. How should I proceed @diasdavid ? A new PR fixing this typo?

Reply:
@vasco-santos yes please.